app/grandchallenge/emails/emails.py

```diff
@@ -1,34 +1,111 @@
 from django.conf import settings
-from django.core.mail import send_mail  # noqa: I251
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMultiAlternatives, get_connection
 from django.template.loader import render_to_string
-from django.utils.html import format_html
+from django.utils.html import format_html, strip_tags
 
+from grandchallenge.profiles.models import EmailSubscriptionTypes
+from grandchallenge.subdomains.utils import reverse
 
-def send_standard_email(*, site, subject, message, recipient, unsubscribable):
-    if not recipient.is_active:
-        # Do not send emails to inactive users
-        return
 
-    if (
-        unsubscribable
-        and not recipient.user_profile.receive_notification_emails
-    ):
-        # Do not send unsubscribable emails to users who have opted out
-        return
+def filter_recipients(recipients, subscription_type=None):
+    filtered_recipients = []
+    for recipient in recipients:
+        if not recipient.is_active:
+            # Do not send emails to inactive users
+            continue
 
-    send_mail(
+        if (
+            subscription_type == EmailSubscriptionTypes.NOTIFICATION
```

Comment:
Maybe we should extend EmailSubscriptionTypes with SYSTEM? Then we can set subscription type to always be required and know that it is being correctly handled.

Reply:
Good idea!